Rakefile

```diff
@@ -184,562 +184,577 @@ task :'release:docs' => :docs do
   sh "aws s3 cp --recursive doc/ s3://#{S3_BUCKET}/#{S3_DIR}/docs/"
 end
 
+# Declare a command for execution.
+# Jobs are parallelized if running in CI.
+def declare(*args)
+  total_executors = ENV.key?('CIRCLE_NODE_TOTAL') ? ENV['CIRCLE_NODE_TOTAL'].to_i : nil
+  current_executor = ENV.key?('CIRCLE_NODE_INDEX') ? ENV['CIRCLE_NODE_INDEX'].to_i : nil
+
+  if total_executors && current_executor && total_executors > 1
+    @execution_count ||= 0
+    @execution_count += 1
+    sh(*args) if @execution_count % total_executors == current_executor
+  else
+    sh(*args)
+  end
+end
+
 desc 'CI task; it runs all tests for current version of Ruby'
 task :ci do
   if Gem::Version.new(RUBY_VERSION) < Gem::Version.new(Datadog::VERSION::MINIMUM_RUBY_VERSION)
     raise NotImplementedError, "Ruby versions < #{Datadog::VERSION::MINIMUM_RUBY_VERSION} are not supported!"
   elsif Gem::Version.new('2.0.0') <= Gem::Version.new(RUBY_VERSION) \
         && Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.1.0')
     # Main library
-    sh 'bundle exec rake test:main'
-    sh 'bundle exec rake spec:main'
-    sh 'bundle exec rake spec:contrib'
+    declare 'bundle exec rake test:main'
```

Comment:
I'm not 100 sold on this name change to `declare`.

I wanted to change it to name that displayed that we are not necessarily running the command inline, the moment it's defined, but that we've informed the testing framework that we'd like to test this command.

Whether it's executed inline, or in a different container is implementation detail.